packages/stark-ui/src/modules/table/entities/column-properties.intf.ts

```diff
@@ -86,4 +86,16 @@ export interface StarkTableColumnProperties {
 	 * Priority of the column. Default: PERSIST
 	 */
 	priority?: StarkTableColumnPriority;
+
+	/**
+	 * Whether the column should be shown on the right side of the table (stickyEnd)
+	 * Default: false
+	 */
+	stickyEnd?: boolean;
```

Comment:
Question: what happens if the developer sets a column as `stickyEnd` but there is another column to the right that is not set as `stickyEnd`? It's not clear what the end result would be (It's not explained neither in the [NG Material docs](https://v7.material.angular.io/components/table/overview#sticky-rows-and-columns)).

Reply:
If it is a sticky column to the left, for example the 3rd column is sticky, when you scroll it behaves as usual. But when the sticky column (3rd one) is going to disappear from the viewport it sticks to the left and stays visible (sticky).
You can see this behaviour in the demo table I've added.

If it's a stickyEnd column to the right (not the last one, but for example the 3rd column from the end)
And there are more columns than the viewport can show, the sticky column will be sticky on the right side.
When you scroll to the right, the sticky column will eventually (when you come to the end) start behaving as usual until you scroll back to the left and it becomes sticky to the right again.

Reply:
Sorry for my late late late reply :disappointed: 
Now it's clear enough, but I would also propose to add the link to the [NG Material docs about Sticky Rows and Columns](https://v7.material.angular.io/components/table/overview#sticky-rows-and-columns) in this comment and also in the one of the `sticky` property so we can have it as reference and better understand how it works. What do you think?